### ISSUE TITLE
Implement support for bibtex files

### ISSUE THREAD

**Issue by angstr0m** [angstr0m/PaperBeard#2]

**Comment by 1blankz7** [comment]
What do you mean with support for bibtex files? Issue #9 allows us to export data given by a search engine to bibtex.

Would you like to search in a already created bibtex file and start requesting a search engine with this informations?

**Comment by angstr0m** [comment]
Exactly that. I figured it would be nice to have that functionality if 
you already have a bibtex library and want the metadata from Google 
Scholar for them.

An export to bibtex would be nice to have.

------ Originalnachricht ------
Von: "Christian Blank" notifications@github.com
An: "angstr0m/PaperBeard" PaperBeard@noreply.github.com
Cc: "angstr0m" malte.eckhoff@googlemail.com
Gesendet: 01.11.2015 18:15:23
Betreff: Re: [PaperBeard] Implement support for bibtex files (#2)

**Comment by 1blankz7** [comment]
Now we have one :-)

Okay, this will mean a move into another direction. I have something like logstash in mind. You have a set of sources (stdin, files), a filter to bring them into a processable format, the search engine, a filter to export the data and a sink to write the data (stdout, file).

**Comment by angstr0m** [comment]
Agreed. The current "architecture" was only meant to deliver quick results so I could continue working on the master thesis.
